pages/apis/webhooks/agent_events.md.erb

```diff
@@ -4,30 +4,7 @@
 
 ## Events
 
-<table>
-<tbody>
-  <tr>
-    <th><code>agent.connected</code></th>
-    <td>An agent has connected to the API</td>
-  </tr>
-  <tr>
-    <th><code>agent.lost</code></th>
-    <td>An agent has been marked as lost. This happens when Buildkite stops receiving pings from the agent.</td>
-  </tr>
-  <tr>
-    <th><code>agent.disconnected</code></th>
-    <td>An agent has disconnected. This happens when the agent shuts down and disconnects from the API.</td>
-  </tr>
-  <tr>
-    <th><code>agent.stopping</code></th>
-    <td>An agent is stopping. This happens when an agent is instructed to stop from the API. It first transitions to stopping and finishes any current jobs.</td>
-  </tr>
-  <tr>
-    <th><code>agent.stopped</code></th>
-    <td>An agent has stopped. This happens when an agent is instructed to stop from the API. It can be graceful or forceful.</td>
-  </tr>
-</tbody>
-</table>
+<%= render_markdown 'apis/webhooks/agent_events_table' %>
```

Comment:
This will break for sure. Assuming including part of a table works, like in https://github.com/buildkite/docs/pull/1295/files#diff-79522a550e85f2b89ca4a63ce7ce748cb182b8277e844d8cfae5179a72c44dcfR26

What you need to do is leave the 

```
<table>
<tbody>
<%= render_markdown 'apis/webhooks/agent_events_table' %>
</tbody>
</table>
```

here, and make sure you're always including the same level of structure.

Reply:
Thanks, @plaindocs! I'll redo the table.